extensions/amp-viqeo-player/OWNERS.yaml

```diff
@@ -2,3 +2,4 @@
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
 - viqeo.tv
+- ampproject/wg-ui-and-a11y
```

Comment:
Is `viquo.tv` even a valid GitHub username? I'm guessing not (probably a vestige of the days before the owners bot was really running) and can probably be dropped.

Reply:
Updated with the correct username from the original PR

Reply:
This was not updated?

Reply:
Oh wow I messed up. Changed `amp-fx-flying-carpet` instead. I'll send a PR now

Reply:
Thanks!